webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js

```diff
@@ -170,6 +187,10 @@ export class SakaiRubricCriteria extends RubricsElement {
           <span tabindex="0" role="button" class="add fa fa-plus"></span>
           <sr-lang key="add_criterion">Add Criterion</sr-lang>
         </button>
+        <button class="add-empty-criterion" @click="${(event) => this.createCriterion(event, true)}">
+          <span tabindex="0" role="button" class="add fa fa-plus"></span>
```

Comment:
Looks like false positive

Reply:
It looks like an ok comment to me.

* Why is the empty icon tabbable? What would a keyboard user do with it?
* What does it have a button role if clicking does nothing?

Reply:
I called this a false positive because on line 192 there is `<sr-lang key="save_weights">Save Weights</sr-lang>`. Which I assumed addresses this. But the tabindex seems out of lace, especially within a button :thinking:...  Again, this is exactly the same pattern, as with the button next to it (the lines above)

Reply:
we will take a look